### ISSUE TITLE
Count of PSF sources

### ISSUE THREAD

**Issue by weaverba137** [legacysurvey/legacysurvey#98]
We have completed a test load of the dr8/south catalog, and again there is a discrepancy in the number of objects, although it is much more subtle this time.  The *total* number of rows in the table does match the number of rows in the tractor files.  *However*:
```
select type, count(*) from tractors_load where brick_primary = 1 group by type;
 type |   count   
~--~--+~--~--~--~-
 COMP |    900231
 DEV  |  43963922
 DUP  |    111172
 EXP  | 117234339
 PSF  | 603839506
 REX  | 532851020
(6 rows)
```
The PSF count reported in http://www.legacysurvey.org/dr8/description/#morphological-classification is 603,773,970, which is exactly 2^16 (65536) less than the count above.  Does anyone know what could cause this?

**Comment by weaverba137** [comment]
Did everyone see this?  Assigning to @dstndstn for now.

**Comment by dstndstn** [comment]
ahhh.... the survey-bricks-{north,south}* summary files use an int16 to hold the number of sources, so that is suspicious.  65k would be a large number of sources for a brick, but I suppose that one of the globular cluster bricks could hit that number.  (I guess I am predicting that exactly one has exceeded that number.)  I will dig.

**Comment by dstndstn** [comment]
(sorry for the delay, I read this when you first sent it but didn't have any ideas)

**Comment by dstndstn** [comment]
Found it -- brick 2296p020 (M5) contains 32,943 PSFs (and no other types), and in FITS's i16 datatype, this becomes -32593, so we're off by 65,536 in this survey-bricks-dr8-south.fits.gz file, for that one entry.

That file is used to produce the summary numbers on the web pages, so we'll need to update that too.  (I'll send a PR, @geordie666 )

Thanks for the persistence, Ben.

**Comment by weaverba137** [comment]
Thank *you* Dustin.  I'll update people on the Data Lab side.

**Comment by weaverba137** [comment]
Will be closed by #99
